src/polyfill.web.tsx

```diff
@@ -1 +1,56 @@
-export * from './mock';
\ No newline at end of file
+import * as React from 'react';
+import { View } from 'react-native';
```

Comment:
Attempt to make this agnostic of RN https://github.com/expo/react-native-appearance/pull/7/files#r334293600